src/cr/cube/matrix/assembler.py

```diff
@@ -149,28 +149,29 @@ def columns_margin(self):
         # --- TODO: Should column_margin only be defined when it's 1D? This would
         # --- require changes to exporter to use the bases to give a
         # --- "column_margin_range"
```

Comment:
I would say yes, same as `rows_margin`. That doesn't mean it needs to be done in this PR, but it's part of the scope of measure consolidation.